start-page/beta/index.html

```diff
@@ -2,39 +2,169 @@
+              </svg>
+              <strong class="uppercase">{{ _('Warning!') }}</strong>
+              {{ _('Beta can be unstable, and there is potential to lose data.') }}
+            </p>
+            <ul>
+              <li>
+                {{ _('Protect your data and other best practices.') }}
+                <a href="https://support.mozilla.org/kb/thunderbird-beta" class="warning">
+                  {{ _('Knowledge Base') }}
+                  <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 12 12" height="12" width="12">
+                    <path d="M4 1C2.347 1 1 2.347 1 4v4c0 1.653 1.347 3 3 3h4c1.653 0 3-1.347 3-3v-.5a.5.5 0 0 0-.5-.5.5.5 0 0 0-.5.5V8c0 1.117-.883 2-2 2H4c-1.117 0-2-.883-2-2V4c0-1.117.883-2 2-2h.5a.5.5 0 0 0 .5-.5.5.5 0 0 0-.5-.5Zm2.5 0a.5.5 0 0 0-.5.5.5.5 0 0 0 .5.5h2.793L6.146 5.146a.5.5 0 0 0 0 .708.5.5 0 0 0 .708 0L10 2.707V5.5a.5.5 0 0 0 .5.5.5.5 0 0 0 .5-.5v-4a.5.5 0 0 0-.094-.29.5.5 0 0 0-.002-.005.5.5 0 0 0-.004-.004.5.5 0 0 0-.046-.055.5.5 0 0 0-.034-.023.5.5 0 0 0-.062-.045.5.5 0 0 0-.065-.035.5.5 0 0 0-.082-.025.5.5 0 0 0-.072-.012A.5.5 0 0 0 10.5 1Z"/>
+                  </svg>
+                </a>
+              </li>
+              <li>
+                {{ _('Add-ons may not work properly, or not work at all.') }}
+                <a href="https://support.mozilla.org/kb/thunderbird-beta#w_add-ons" class="warning">
+                  {{ _('Add-ons Guide') }}
+                  <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 12 12" height="12" width="12">
+                    <path d="M4 1C2.347 1 1 2.347 1 4v4c0 1.653 1.347 3 3 3h4c1.653 0 3-1.347 3-3v-.5a.5.5 0 0 0-.5-.5.5.5 0 0 0-.5.5V8c0 1.117-.883 2-2 2H4c-1.117 0-2-.883-2-2V4c0-1.117.883-2 2-2h.5a.5.5 0 0 0 .5-.5.5.5 0 0 0-.5-.5Zm2.5 0a.5.5 0 0 0-.5.5.5.5 0 0 0 .5.5h2.793L6.146 5.146a.5.5 0 0 0 0 .708.5.5 0 0 0 .708 0L10 2.707V5.5a.5.5 0 0 0 .5.5.5.5 0 0 0 .5-.5v-4a.5.5 0 0 0-.094-.29.5.5 0 0 0-.002-.005.5.5 0 0 0-.004-.004.5.5 0 0 0-.046-.055.5.5 0 0 0-.034-.023.5.5 0 0 0-.062-.045.5.5 0 0 0-.065-.035.5.5 0 0 0-.082-.025.5.5 0 0 0-.072-.012A.5.5 0 0 0 10.5 1Z"/>
+                  </svg>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <p class="beta-thanks">{{ _('Thank you for using Thunderbird beta. Your experiences with beta are important, so please report rough edges, bugs, and provide feedback in the ways suggested below - to help make the next Thunderbird release as awesome as possible.')}}</p>
```

Comment:
`Thunderbird beta` should be `Thunderbird Beta`. When it's not used as a proper name, then it's OK for it to be lowercase.